### PR TITLE
#393 Adding Poker Generator

### PR DESCRIPTION
**Pull request by roman-shuhov**
Generated tests and fixed example implementation to pass the test

### CODE REVIEW

File: exercises/poker/Example.fs [hunk 1]
```diff
@@ -166,7 +170,8 @@ let parsePokerHand (input: string) =
 let bestHands hands =
     let pokerHands = List.map (fun hand -> hand, parsePokerHand hand) hands
     let bestHand = pokerHands |> List.map snd |> List.max 
-    
+    printf "%O" hands
```
Comment (ErikSchierboom) [inline exercises/poker/Example.fs]: Could you remove this line?

File: exercises/poker/Example.fs [hunk 2]
```diff
@@ -166,7 +170,8 @@ let parsePokerHand (input: string) =
...
+    printf "%O\n\n" bestHand
```
Comment (ErikSchierboom) [inline exercises/poker/Example.fs]: Could you remove this line?

**Comment by ErikSchierboom** [comment]
Looking great! Just two small nits.

**Comment by ErikSchierboom** [comment]
Brilliant. Merged! :tada:

**Comment by roman-shuhov** [comment]
Thanks for catching this issue, removed.

**Pull request merged**